docs/assets/2.19-2.20-cleanup-monitoring.sh

```diff
@@ -0,0 +1,120 @@
+kubectl delete -n kyma-system services monitoring-operator --ignore-not-found
+kubectl delete -n kyma-system services monitoring-prometheus --ignore-not-found
+kubectl delete -n kyma-system services monitoring-prometheus-istio-server --ignore-not-found
+kubectl delete -n kyma-system services monitoring-prometheus-node-exporter --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-alertmanager --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-auth-proxy-grafana --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-grafana --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-kube-state-metrics --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-operator --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-prometheus --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-prometheus-istio-server --ignore-not-found
+kubectl delete -n kyma-system serviceaccounts monitoring-prometheus-node-exporter --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-alertmanager --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-apiserver --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-grafana --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-kube-state-metrics --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-kubelet --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-operator --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-prometheus --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-prometheus-istio-server-server --ignore-not-found
+kubectl delete -n kyma-system servicemonitors.monitoring.coreos.com monitoring-prometheus-node-exporter --ignore-not-found
+kubectl delete -n kyma-system virtualservices.networking.istio.io monitoring-grafana --ignore-not-found
+kubectl delete -n kyma-system persistentvolumeclaims prometheus-monitoring-prometheus-db-prometheus-monitoring-prometheus-0 --ignore-not-found
+kubectl delete -n kyma-system configmaps istio-control-plane-grafana-dashboard --ignore-not-found
+kubectl delete -n kyma-system configmaps istio-mesh-grafana-dashboard --ignore-not-found
+kubectl delete -n kyma-system configmaps istio-performance-grafana-dashboard --ignore-not-found
+kubectl delete -n kyma-system configmaps istio-service-grafana-dashboard --ignore-not-found
+kubectl delete -n kyma-system configmaps istio-workload-grafana-dashboard --ignore-not-found
+kubectl delete -n kyma-system configmaps function-metrics-dashboard --ignore-not-found
```

Comment:
There is a service 'monitoring-kubelet' in kube-system generated by the prometheus-operator, will that be deleted by the operator?

The pv for prometheus needs to get deleted explicitly. Not sure how to do that best via script.
What is with the cluster-wide resource like Clusterrole?